### ISSUE TITLE
Promisify localforage.keys() 

### ISSUE THREAD

**Issue by iamolivinius** [mozilla/localForage#532]
Is there any reason why `.keys()` does not provide a promisified API like the other methods?

**Comment by thgreasi** [comment]
I think that [it already returns a promise](https://github.com/mozilla/localForage/blob/master/src/drivers/indexeddb.js#L738)!?!?!?

**Comment by tofumatt** [comment]
Yeah, it should return a promise. Maybe the docs are out of date?
- tofumatt

**Comment by iamolivinius** [comment]
Ah, I see. Just read the docs but the actual implementation looks fine. Sorry for the noise. :blush:

**Comment by tofumatt** [comment]
No worries! I'll check on the docs and fix them before I close this issue
:-)
- tofumatt

**Comment by fjmorel** [comment]
All 3 drivers support Promises and the docs don't say otherwise.
